packages/cspell/src/app/lint/lint.test.ts

```diff
@@ -33,6 +40,79 @@ describe('Linter Validation Tests', () => {
         expect(rWithFiles.files).toBe(1);
     });
 
+test('uses CSPELL_CONFIG_PATH as fallback when config not provided', async () => {
+        const options = { root: environmentSamples };
+        const reporter = new InMemoryReporter();
+        const customConfig = j(samples, 'cspell.custom.json');
+
+        process.env[environmentKeys.CSPELL_CONFIG_PATH] = customConfig;
+
+        const result = await runLint(new LintRequest(['**/*.txt'], options, reporter));
+        expect(result.errors).toBe(0);
+        expect(result.files).toBeGreaterThan(0);
```

Comment:
We should check the number of spelling issues. The config file can have a strong impact on the number of issues found. 
```suggestion
        expect(result.errors).toBe(0);
        expect(result.files).toBeGreaterThan(0);
        expect(result.issues).toBe(0);
```

I used `0`, but it can be a different value depending upon what was checked.

Please add an issues check for each of the tests.